internal/usage/usage.go

```diff
@@ -392,12 +392,12 @@ dbName and collection are required only for built-in roles.`
 	ReadConcern                               = "Default level of acknowledgment requested from MongoDB for read operations set for this cluster."
 	WriteConcernAdvancedSettings              = "Default level of acknowledgment requested from MongoDB for write operations set for this cluster."
 	TLSProtocol                               = "Minimum Transport Layer Security (TLS) version that the cluster accepts for incoming connections."
-	DisableTableScan                          = "Flag that disables executing any query that requires a collection scan to return results."
-	EnableTableScan                           = "Flag that enables executing any query that requires a collection scan to return results."
-	DisableFailIndexKeyTooLong                = "Flag that disables the mongod to writes documents that exceed 1024 bytes but doesn't index them."
-	EnableFailIndexKeyTooLong                 = "Flag that enables the mongod to write documents that exceed 1024 bytes limit but doesn't index them."
-	DisableJavascript                         = "Flag that disables the execution of operations that perform server-side executions of JavaScript."
-	EnableJavascript                          = "Flag that enables the execution of operations that perform server-side executions of JavaScript."
+	DisableTableScan                          = "Disables executing any query that requires a collection scan to return results."
+	EnableTableScan                           = "Enables executing any query that requires a collection scan to return results."
+	DisableFailIndexKeyTooLong                = "Disables writing documents that exceed 1024 bytes without indexing."
+	EnableFailIndexKeyTooLong                 = "Enables writing documents that exceed 1024 bytes without indexing."
```

Comment:
These lines have the meaningful changes.

Is this an accurate description of behavior?